pkg/run/watch/forward_port.go

```diff
@@ -126,3 +134,40 @@ func ForwardPort(log logr.Logger, pod *corev1.Pod, cfg *rest.Config, specMap *Po
 
 	return fw.ForwardPorts()
 }
+
+func ShowPortForwards(log clilogger.Logger, portForwards map[string]*PortForwardShortcut) {
```

Comment:
```suggestion
func ShowPortForwards(log clilogger.Logger, portForwards []PortForwardShortcut) {
```

Reply:
And here should it be a slice or a map[rune]portforward?